app/gateway/rate-limiting/strategies.md

```diff
@@ -10,6 +10,10 @@ plugins:
   - rate-limiting-advanced
   - ai-rate-limiting-advanced
   - graphql-rate-limiting-advanced
+
+breadcrumbs:
+  - /gateway/
+  - /gateway/rate-limiting/
```

Comment:
@lena-larionova I wasn't sure about these rate-limiting ones, WDYT? We can always re-visit them when we migrate the content